addons/l10n_be/data/account_fiscal_position_tax_template_data.xml

```diff
@@ -152,11 +152,6 @@
             <field name="tax_dest_id" ref="attn_VAT-OUT-00-ROW"/>
         </record>
         <record id="afpttn_extracom_7" model="account.fiscal.position.tax.template">
-            <field name="position_id" ref="fiscal_position_template_2"/>
-            <field name="tax_src_id" ref="attn_VAT-OUT-21-S"/>
-            <field name="tax_dest_id" ref="attn_VAT-OUT-00-ROW"/>
-        </record>
-        <record id="afpttn_extracom_8" model="account.fiscal.position.tax.template">
```

Comment:
Diff is weird: you're removing afpttn_extracom_8 and replacing afpttn_extracom_7's content. I'd simply keep afpttn_extracom_8 and remove afpttn_extracom_7, just for the sake of history readability.

Reply:
Indeed, don't know why I did that :sweat_smile: